host-configs/lassen-blueos_3_ppc64le_ib_p9-clang@10.0.1.1.cmake

```diff
@@ -101,19 +101,19 @@ set(CAMP_DIR "${TPL_ROOT}/camp-2022.03.2" CACHE PATH "")
 
 # Root directory for generated developer tools
 
-set(DEVTOOLS_ROOT "/collab/usr/gapps/axom/devtools/blueos_3_ppc64le_ib_p9/2022_07_12_11_54_06/gcc-8.3.1" CACHE PATH "")
+set(DEVTOOLS_ROOT "/collab/usr/gapps/axom/devtools/blueos_3_ppc64le_ib_p9/2023_04_18_13_41_48/._view/srxt35kojgk77f2222mk6mgv7z5jyyzz" CACHE PATH "")
```

Comment:
For the future, should we try to move the output up above the hidden `.view` directory?

Reply:
I think this is a duplicate directory that Spack is creating.. there is also a straight `view` directory right under the timedate stamp directory.

We could investigate later if we can remove that duplication.